Build fails on armv7

Since https://github.com/moby/moby/pull/38050 our nightly builds for Docker CE have been failing with:

```
Install rootlesskit version 3c4582e950e3a67795c2832179c125b258b78124
+ echo 'Install rootlesskit version 3c4582e950e3a67795c2832179c125b258b78124'
+ git clone https://github.com/rootless-containers/rootlesskit.git /tmp/tmp.YNt7AorSyx/src/github.com/rootless-containers/rootlesskit
Cloning into '/tmp/tmp.YNt7AorSyx/src/github.com/rootless-containers/rootlesskit'...
+ cd /tmp/tmp.YNt7AorSyx/src/github.com/rootless-containers/rootlesskit
+ git checkout -q 3c4582e950e3a67795c2832179c125b258b78124
+ go build -ldflags= -o /build//rootlesskit github.com/rootless-containers/rootlesskit/cmd/rootlesskit
# github.com/rootless-containers/rootlesskit/pkg/network/iputils
pkg/network/iputils/iputils.go:18:12: constant 4294967295 overflows int
```

This error only occurs when we're running on armv7 (arm32) machines.

CC @AkihiroSuda

--- a/pkg/network/iputils/iputils.go
+++ b/pkg/network/iputils/iputils.go
@@ -14,11 +14,11 @@ func AddIPInt(ip net.IP, i int) (net.IP, error) {
 		return nil, errors.Errorf("expected IPv4 address, got %s", ip.String())
 	}
 	ui32 := binary.BigEndian.Uint32(ip)
-	resInt := int(ui32) + i
-	if resInt > math.MaxUint32 {
+	resInt64 := int64(ui32) + int64(i)
+	if resInt64 > int64(math.MaxUint32) {
 		return nil, errors.Errorf("%s + %d overflows", ip.String(), i)
 	}
 	res := make(net.IP, 4)
-	binary.BigEndian.PutUint32(res, uint32(resInt))
+	binary.BigEndian.PutUint32(res, uint32(resInt64))
 	return res, nil
 }

--- a/pkg/version/version.go
+++ b/pkg/version/version.go
@@ -1,3 +1,3 @@
 package version
 
-const Version = "0.3.0-alpha.1"
+const Version = "0.3.0-alpha.1+dev"

--- a/pkg/version/version.go
+++ b/pkg/version/version.go
@@ -1,3 +1,3 @@
 package version
 
-const Version = "0.3.0-alpha.0+dev"
+const Version = "0.3.0-alpha.1"

